src/hermes.jsx

```diff
@@ -305,6 +307,7 @@ module.exports = React.createClass({
 	},
 
 	onBlur: function () {
+		this.clearSuggestions();
```

Comment:
arrowing down does not lose focus, but with this line clicking away does (and without it clicking away does nothing)